package.json

```diff
@@ -69,6 +69,7 @@
     }
   },
   "dependencies": {
+    "@regosen/gapless-5": "file:regosen-gapless-5-1.5.6.tgz",
```

Comment:
Temporary until https://github.com/regosen/Gapless-5/pull/59 is merged (hopefully)